src/plugins/intel_gpu/tests/functional/shared_tests_instances/behavior/ov_plugin/properties_tests.cpp

```diff
@@ -701,8 +700,6 @@ TEST_P(OVGetMetricPropsTest_CACHING_PROPERTIES, GetMetricAndPrintNoThrow) {
         ASSERT_TRUE(std::find(caching_properties.begin(), caching_properties.end(), property_name) !=
                     caching_properties.end());
     }
-
-    OV_ASSERT_PROPERTY_SUPPORTED(ov::internal::caching_properties);
```

Comment:
I think this might be a mistake?

Reply:
Reviewed again, expectation behavior.
https://github.com/openvinotoolkit/openvino/blob/3236ff3f20b7e00a89aa1ee725682b9f0fccc5c7/src/plugins/intel_gpu/src/plugin/plugin.cpp#L562

Reply:
I'm not sure but @nkogteva @riverlijunjie seems to know related cache properties.
Could you let me know or confirm about `caching_properties` is in `internal_properties` NOT `properties`. Is it intended behavior? 
I don't have background about cache properties and now function test issue was occurred, need to confirms.